### PR TITLE
fixes #2600 (usage of obsolete pipe model)

### LINKED ISSUE #2600: Modelica.Electrical.Machines.Examples.DCMachines.DCPM_Cooling uses obsolete pipe model
Dymola generates the warning:

Checking model Modelica.Electrical.Machines.Examples.DCMachines.DCPM_Cooling:
Warning: Component type specifier HeatedPipe specified an obsolete type: Obsolete model - use Modelica.Thermal.FluidHeatFlow.Components.Pipe(useHeatPort=true) instead.

I can understand that we test obsolete models, but not that we use them in non-obsolete-examples.

### PR DESCRIPTION
**Pull request by AHaumer**
Two issues fixed by one pull request:
close #2600 It wasn't intended to leave the obsolete pipe model in this examples

### CODE REVIEW

File: Modelica/Electrical/Machines.mo [hunk 1]
```diff
@@ -4713,6 +4713,7 @@ So the machine is at the beginning in cold condition, ending in warm condition
       end DCPM_Temperature;
 
       model DCPM_Cooling "Test example: Cooling of a DCPM motor"
+        import Modelica;
```
Comment (beutlich) [inline Modelica/Electrical/Machines.mo]: We want to get rid of useless imports.

Comment (AHaumer) [reply Modelica/Electrical/Machines.mo]: Done by my last commit/push to my fork/branch

**Comment by beutlich** [comment]
OK, PR is updated to only consider #2600.

**Comment by AHaumer** [comment]
I don't understand "duplicate icon-change".
If the icon is already changed, why can't I see this in upstream/master?

**Comment by beutlich** [comment]
> If the icon is already changed, why can't I see this in upstream/master?

Because #2621 misses your review.

**Pull request merged**